src/main/resources/default/templates/biz/process/process.html.pasta

```diff
@@ -89,16 +89,18 @@
                 </i:for>
             </t:infobox>
 
-            <i:if test="@!process.getOutputs().isEmpty()">
+            <i:if test="@process.hasVisibleOutputs()">
                 <t:navbox labelKey="Process.outputs">
                     <t:navboxLink icon="fa fa-bars" labelKey="ProcessLog.plural"
                                   url="@apply('/ps/%s', process.getId())"
                                   active="page == 'logs'"/>
 
                     <i:for type="sirius.biz.process.output.ProcessOutput" var="output" items="@process.getOutputs()">
-                        <t:navboxLink icon="@output.getIcon()" label="@output.getLabel()"
-                                      url="@apply('/ps/%s/output/%s', process.getId(), output.getName())"
-                                      active="page == 'output-' + output.getName()"/>
+                        <i:if test="@!output.isSystemOutput() || user().hasPermission('permission-manage-all-processes')">
```

Comment:
shouldn't we also enforce this check when actually rendering the output (like in the controller)?